src/MaaWpfGui/ViewModels/UI/RecognizerViewModel.cs

```diff
@@ -623,8 +647,8 @@ public bool OperBoxParse(JObject details)
         /// <summary>
         /// 开始识别干员
         /// </summary>
-        // xaml 中用到了
-        // ReSharper disable once UnusedMember.Global
+        /// xaml 中用到了
+        /// ReSharper disable once UnusedMember.Global
```

Comment:
This is not a documentation note, use two '/'

Reply:
The warning pissed me off, my bad